include/uv-unix.h

```diff
@@ -357,13 +352,7 @@ typedef struct {
   int mode;
 
 #define UV_SIGNAL_PRIVATE_FIELDS                                              \
-  /* RB_ENTRY(uv_signal_s) tree_entry; */                                     \
-  struct {                                                                    \
-    struct uv_signal_s* rbe_left;                                             \
-    struct uv_signal_s* rbe_right;                                            \
-    struct uv_signal_s* rbe_parent;                                           \
-    int rbe_color;                                                            \
-  } tree_entry;                                                               \
+  RB_ENTRY(uv_signal_s) tree_entry;                                            \
```

Comment:
The reason I didn't use RB_ENTRY is that it conflicts with the macro of the same name on BSD-like (and Solaris-like?) systems.  It's defined in `<sys/tree.h>` and people tend to include that file in their projects along with uv.h.  That's also the reason why our tree.h isn't included in uv.h or uv-unix.h.